etc/frontend/sections.xml

```diff
@@ -43,4 +43,11 @@
         <section name="boltcart"/>
     </action>
     <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <action name="rest/*/V1/guest-carts/*/gift-message">
+        <section name="boltcart"/>
```

Comment:
@johnmccombs1 Could you please explain in which cases those AJAX calls happen?

Reply:
@vitaliyreznikov It’s when the customer applies a gift message on the cart page. See here: http://prntscr.com/t96tqa